tutorials/text_to_speech.ipynb

```diff
@@ -0,0 +1,283 @@
+                "play_audio(audios[0])"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Even more examples\n",
+                "Let's now try the advanced ```Mars5-tts``` model.\n",
+                "\n",
+                "```Mars5-tts``` requires two inputs:\n",
+                "1. A list of pieces of text you want to synthesize.\n",
+                "2. Target voices you want to clone, along with their respective transcripts.\n",
+                "Although transcripts are not strictly necessary for the model to function, our initial tests show that they significantly improve the model's quality. For this reason, we have made transcripts mandatory in our interface in ```senselab```."
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### Setup\n",
+                "First, let's import the necessary libraries and the function we'll be using."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "from senselab.utils.data_structures.model import TorchModel\n",
+                "from senselab.utils.data_structures.language import Language\n",
+                "from senselab.utils.data_structures.device import DeviceType\n",
+                "from senselab.audio.data_structures.audio import Audio\n",
+                "from senselab.audio.tasks.preprocessing.preprocessing import resample_audios, downmix_audios_to_mono, extract_segments\n",
+                "from senselab.audio.tasks.plotting.plotting import play_audio\n",
+                "from senselab.audio.tasks.text_to_speech import synthesize_texts"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### Specifying the TTS model, the language and the preferred device\n",
+                "Let's initialize the model we want to use (remember to specify both the ```path_or_uri``` and the ```revision``` for reproducibility purposes), the language of the text we want to synthetize, and the device we prefer. In this tutorial, we are going to use [```mars5```](https://github.com/Camb-ai/MARS5-TTS), which only works for English."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "model = TorchModel(path_or_uri=\"Camb-ai/mars5-tts\", revision=\"master\")\n",
+                "language = Language(language_code=\"en\")\n",
+                "device = DeviceType.CPU"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
```

Comment:
targets is supposed to be a tuple, not a list. code errors.

Reply:
I get the same error. Line 239